### ISSUE TITLE
parameter setting

### ISSUE THREAD

**Issue by HongboBo** [xptree/DeepInf#6]
Hello, I am really interested in your research. 

May I ask the parameters setting of your code, which can get the same best results in your paper 'DeepInf: Social Influence Prediction with Deep Learning'. Especially on your twitter dataset.

Many thanks.

**Comment by xptree** [comment]
Could you provide more details? What hyper-parameters are you using? And what are the results?

**Comment by HongboBo** [comment]
Many thanks for your reply!

The hyper-parameters I used:
epochs:1000
learning rate: 1e-3
dropout: 0.2
hidden units:16, 8
gat heads in each layer:1,1,1
embedding dimension:64
class-weight-balanced: false
weight decay for  Adagrad optimizer: 5e-4

Data set: Twitter
The results:
![image](https://user-images.githubusercontent.com/46356325/88591532-65aeb480-d054-11ea-8bfc-e22403db2ec6.png)

**Comment by xptree** [comment]
Hi, it seems that you use different hyper-parameters than those reported in the paper.

Could you check section 4.3 of https://arxiv.org/pdf/1807.05560.pdf for more details? There is a subsection titled `Hyper-parameter Setting & Implementation Details`.

Thanks.

**Comment by HongboBo** [comment]
I tried it and got similar results to your paper.
Thanks a lot.
